src/Avalonia.Controls.TreeDataGrid/Models/TreeDataGrid/TextColumnOptions.cs

```diff
@@ -13,6 +15,16 @@ public class TextColumnOptions<TModel> : ColumnOptions<TModel>, ITextCellOptions
         /// </summary>
         public bool IsTextSearchEnabled { get; set; }
 
+        /// <summary>
+        /// Gets or sets the format string for the cells in the column.
+        /// </summary>
+        public string StringFormat { get; set; } = "{0}";
```

Comment:
I just saw that we have the `TModel` here! Might be able to use `StringSyntaxAttribute` after all?